Dockerfile

```diff
@@ -56,6 +56,7 @@ RUN apt-get update
 RUN apt-get install -y ruby2.3 
 RUN apt-get install -y ruby2.3-dev
 RUN gem install bundler
+RUN apt-get install build-essential
```

Comment:
Is this ok here?  You don't need it before installing bundler?  I guess bundler probably doesn't have any native extensions, so might be ok.